pyvo/dal/query.py

```diff
@@ -894,6 +894,9 @@ def make_dataset_filename(self, *, dir=".", base=None, ext=None):
         if not ext:
             ext = self.suggest_extension(default="dat")
 
+        base = base.replace("/", "_"
+            ).replace("\\", "_")
+
```

Comment:
1. I would replace each `\` with `_` to correctly keep track of the replacements. Single `\` might also create issues.
2. I've noticed that `suggest_dataset_basename` fixes empty spaces too but this is not the case if `base` is present. Should we do this sanitization in one place (and potentially add more rules to it over time)?

Reply:
ahh, I'm sorry for the premature merge then.